integrationExamples/gpt/pbjs_example_gpt.html

```diff
@@ -343,6 +343,13 @@
                         params: {
                             inScreen: 'ggumtest' // REQUIRED str Tracking Id
                         }
+                    },
+                    {
+                        bidder: 'wideorbit',
+                        params: {
+                            pbId: 123, // REQUIRED Publisher Id,
+                            pId: 123456 // REQUIRED Placement Id
+                        }

```

Comment:
Thank you for spotting this. It looks like there was a problem when mergin in the latest code.